### PR TITLE
KeyDown/KeyUp map to unknown and call event

### PR DESCRIPTION
**Pull request by usertoroot**
# Summary of the PR
Instead of not calling KeyDown when the key is not found, call it with Key.Unknown and the platform specific scan code.

# Related issues, Discord discussions, or proposals
[Issue 846](https://github.com/dotnet/Silk.NET/issues/846)

# Further Comments
Perhaps as a next step the scan codes between GLFW and SDL should be made the same.

### CODE REVIEW

File: src/Input/Silk.NET.Input.Sdl/SdlKeyboard.cs [hunk 1]
```diff
@@ -40,20 +40,40 @@ public unsafe void DoEvent(Event @event)
...
-                        KeyUp?.Invoke(this, key, (int) @event.Key.Keysym.Scancode);
+                        Key keyUp;
+                        if (_keyMap.TryGetValue((KeyCode) @event.Key.Keysym.Sym, out var key))
+                        {
+                            keyUp = key;
+                            _keysDown.Remove(keyUp);
+                        }
+                        else
+                        {
+                            keyUp = key;
```
Comment (ThomasMiz) [inline src/Input/Silk.NET.Input.Sdl/SdlKeyboard.cs:73]: Did you mean `keyUp = Keys.Unknown;` in here?

**Review by ThomasMiz — APPROVED** [review]
Looks good to me!

**Comment by usertoroot** [comment]
@ThomasMiz Seems like the CI Build failed on installing workload:
```
Workload installation failed: Workload not found: android-aot. Known workloads: android android-33 ios maccatalyst macos maui maui-mobile maui-desktop maui-core maui-blazor maui-android maui-maccatalyst maui-ios maui-windows tvos microsoft-net-sdk-emscripten wasm-tools microsoft-net-runtime-android microsoft-net-runtime-android-aot microsoft-net-runtime-ios runtimes-ios microsoft-net-runtime-maccatalyst runtimes-maccatalyst microsoft-net-runtime-macos microsoft-net-runtime-tvos runtimes-tvos microsoft-net-runtime-mono-tooling
```

Do you want me to commit to re-trigger the CI Build to run?

**Comment by Perksey** [comment]
How frustrating, as if they just make breaking changes like that! We can just merge anyway, I’ll do that in a bit as this isn’t allowed by default.

**Comment by usertoroot** [comment]
@Perksey Thanks I appreciate it!

**Pull request merged**